src/utils/saml.ts

```diff
@@ -68,7 +121,7 @@ export const getPreValidateResponse = (
   doneCb,
   callback
   // eslint-disable-next-line sonarjs/cognitive-complexity
-): Promise<void | E.Left<void> | E.Right<void>> => {
+): void => {
```

Comment:
the type `PreValidateResponseT` explicitly says that this function is returning void, indicating that the return edvalue is not used. I procedeed and edited this as a `void` returning function, as the type suggests.